cmake-modules/AzureDoxygen.cmake

```diff
@@ -38,6 +38,10 @@ function(generate_documentation PROJECT_NAME PROJECT_VERSION)
         # Use MathJax instead of latex to render formulas
         set(DOXYGEN_USE_MATHJAX YES)
 
+        set(DOXYGEN_REPEAT_BRIEF NO)
+        #set(DOXYGEN_ALWAYS_DETAILED_SEC NO)
+        #set(DOXYGEN_BRIEF_MEMBER_DESC YES)
```

Comment:
Add comments into this file for why we are setting certain values.  

Is there a reason to keep the two options that are commented out?